handbook/ldap/ldap.md

```diff
@@ -0,0 +1,120 @@
+
+- An LDAP server accessible to the SystemLink server.
+
+- A bind user and bind password for the LDAP server.
+
+- Familiarity with the LDAP attributes available to your organization.
+    - If you do not know what LDAP attributes and groups are available to you, talk with your LDAP system administrator.
+    - Tools such [ADExplorer](https://docs.microsoft.com/en-us/sysinternals/downloads/adexplorer) can be helpful to explore the attributes assigned to your users.
+
+## Enabling LDAP in SystemLink
+
+### Configuring Authentication
+
+1. Log into the server running SystemLink and open **NI Web Sever Configuration**.
+
+2. Go to the **Authentication** tab and enable **Connect to an LDAP server**.
+
+3. Enter the LDAP URL for your sever. Refer to [**LDAP URLs**](#ldap-url) for details on how to structure this URL.
+
+4. Enter the bind user. Refer to [**Bind User**](#bind-user) for details on usernames that can be used here.
+
+5. Enter the bind password.
+
+6. Click **Apply and Restart**.
+
+<figure>
+  <img src="../../img/ldap-setup.png" width="500" />
+  <figcaption>Configuration needed to enable logging into a SystemLink server using LDAP credentials.</figcaption>
+</figure>
+
+You may now login to the SystemLink server using your LDAP credentials. To access systems and data in SystemLink, configure workspace and role mappings. Refer to [**Mapping LDAP Attributes and Groups to SystemLink Workspaces and Roles**](#mapping-ldap-attributes-and-groups-to-systemlink-workspaces-and-roles) for details.
+
+<figure>
+  <img src="../../img/no-mapping.png" width="500" />
+  <figcaption>View from a user who can login but does not have any workspace and role mappings.</figcaption>
+</figure>
+
+## LDAP URL and Bind User
+
+You must provide a URL and bind user and bind password to establish an authenticated connection between your SystemLink server and LDAP server.
+
+### LDAP URL
+
+The LDAP URL follows a standard scheme. The URL determines the server to connect to, the server's port, a base search distinguished name (DN), and the attribute that determines the SystemLink username for login.
+
+!!! note "Basic LDAP URL Scheme"
+
+    **Structure:** `ldap://<server-dns>:<port>/<target-entry-dn>?<username-attribute>`
+
+    **Example:** `ldap://example.com:389/dc=example,dc=com?sAMAccountName`
+
+    In this example the server is `example.com`, the port is 389, the base search DN is `dc=example,dc=com` and the LDAP attribute used for user login is `sAMAccountName`. 
+
+!!! note "Example LDAP URLs specifying different usernames for the user Jane Doe"
+
+    **URL:** `ldap://example.com/dc=example,dc=com?sAMAccountName`
+    **Username:** jdoe
+
+    **URL:** `ldap://example.com/dc=example,dc=com?userPrincipalName`
+    **Username:** jane.doe@example.com
+
+    <figure>
+      <img src="../../img/ldap-login.png" width="500" />
+      <figcaption>Username `jdoe` is used because the `sAMAccountName` attribute is specified in the LDAP URL.</figcaption>
```

Comment:
It could be worth mentioning that `sAMAccountName` is related to Windows Active Directory and [refers to the Windows username](https://docs.microsoft.com/en-us/windows/win32/adschema/a-samaccountname). Other LDAP servers may not have that attribute.

Reply:
I'm going to leave this out for now, because I don't have a great place to stick this information. We can definitely change this decision later is need be.